examples/OneDShallowWaterGeostrophicAdjustment.jl

```diff
@@ -1,11 +1,11 @@
-# This example solves the 1D rotating shallow water equations
+# This example solves the linear 1D rotating shallow water equations
 # for the depth-integrated velocities u and v and for the
 # surface elevation η in a fluid with constant depth h,
 # gravitational acceleration g and Coriolis parameter f:
 #
-# ∂u/∂t + u∂u/∂x - fv = -gh∂η/∂x - D,   (1)
-# ∂v/∂t + u∂v/∂x + fu = - D,            (2)
-# ∂η/∂t + ∂u/∂x = - D.                  (3)
+# ∂u/∂t - fv = -gh∂η/∂x - D,   (1)
+# ∂v/∂t + fu = - D,            (2)
+# ∂η/∂t + ∂u/∂x = - D.         (3)
```

Comment:
Is this correct? Dimensionally it doesn't make sense since dim(∂η/∂t) = L/T and dim(∂u/∂x) = 1/T.

Reply:
Also term -gh∂η/∂x is not dimensionally consistent with the rest in eq (1). What's η supposedly, isn't it a length?

Reply:
The velocities are vertically integrated, so they have dimensions of L^2/T, so dim(∂u/∂x) = L/T.

All terms in eq. (1) have dimensions of L^2/T^2.